snare/tests/test_add_meta_tag.py

```diff
@@ -21,7 +21,8 @@ def setUp(self):
     def test_add_meta_tag(self):
         config = configparser.ConfigParser()
         config['WEB-TOOLS'] = dict(google='test google content', bing='test bing content')
-        add_meta_tag(self.page_dir, self.index_page, config)
+        default_path = '/opt/snare'
```

Comment:
for test I'd better use smth as tmp

Reply:
@afeena can you elaborate on what are you trying to suggest? I didn't get you.